doc/content/lfr/cardinal_7pincell/Cardinal_7pin_LFR_demo.md

```diff
@@ -0,0 +1,438 @@
+
+### Conservation history style=font-size:125%
+
+[convergence] illustrates the simulation history. 
+The NekRS standalone calculation to generate initial conditions occupies the first 9.016 seconds. Once the fully coupled calculation is initiated, the CMFD + DFEM-$S_N$ calculations continued until the relative Richardson error hit the tolerance of $10^{-3}$. 
+Then, H.C. + NekRS calculations were performed until the end time of H.C. was met. The L2-norm of the coolant temperature field was first calculated then, and the next set of CMFD + DFEM-$S_N$ calculations were performed, which dropped the relative Richardson errors down to $10^{-5}$. 
+After the next cycle of H.C. + NekRS calculations, the L2-norm of the coolant temperature field was updated and its relative difference to its previous value was computed. This process was repeated until the relative change of the L2-norm of the coolant temperature field dropped below $10^{-5}$. 
+Even though the relative Richardson error in Griffin jumped at each subsequent Picard iteration, the overall error as well as the eigenvalue difference decreased with increasing number of Picard iterations.
+
+!media lfr/cardinal_7pin/convergence.png
+       style=width:100%
+       id=convergence
+       caption=Convergence history of the whole simulation ($l_R$ - Richardson iteration index, $l_P$ - Picard iteration index.)
+
+Scheme (A) in [its] resulted in false temperature convergence unless the Richardson tolerance was very carefully controlled. This false convergence arises due to the weak coupling nature of temperature and angular flux in this problem, where monitoring the angular flux convergence is not a good proxy for temperature convergence. 
+[power] shows that the power distribution is almost converged with less than 1% error in only one Picard iteration, despite the temperature not being converged. Although not shown, errors in power distribution kept decreasing after the first Picard iteration. This confirms that neutronics is weakly coupled to temperature in this problem, and that neutronics power distributions from early in the simulation will result in an accurate temperature profile.
+
+!media lfr/cardinal_7pin/power.png
+       style=width:80%
+       id=power
+       caption=Fast convergence of power distribution due to weak coupling.
+
+One noteworthy observation was that infrequent data communication between H.C. and NekRS may result in solution instability. 
+[bugs] shows that when H.C. is called 10 times less frequently (500:1 rather than 50:1 NekRS:H.C.), a large unphysical oscillation appears in the solution.
+Although only the duct heat flux is shown, other solutions also diverged. 
+This observation is consistent with the fact that the operator splitting method is first order accurate in time, and such an observation would occur with other codes selected for the fluid solve and the solid solve (i.e this effect is not specific to Cardinal or MOOSE).
```

Comment:
stability is not the same concept as accuracy, you cant blame the order of accuracy for a stability issue (at least unless you do von neumann analysis and tie the two together, but by default they would not be)

central differencing would be second order accurate, but also most likely inconditionally unstable here

Reply:
Agreed. A lot of the content was copy-pasted from an existing paper. I will try to rephrase it.